transformer_engine/jax/cpp_extensions.py

```diff
@@ -19,6 +19,8 @@
 
 import transformer_engine_jax
 from transformer_engine_jax import DType as TEDType
+from transformer_engine_jax import NVTE_Bias_Type
```

Comment:
Note: This file is partially annotated

Reply:
Annotation added in https://github.com/NVIDIA/TransformerEngine/pull/175/commits/bd8759a189abb5f1748d9c19f85368dd420592e7